### ISSUE TITLE
How to load your pre-trained model and train on my dataset?

### ISSUE THREAD

**Issue by YHDASHEN** [SlongLiu/query2labels#4]
Hi,

thank you so much for your great work. I'm doing a project with multi-label classification so I wonder how I can apply your pretrained model for image feature extraction? what I need is to extract feature of an image. Could you please give me some hints?

Best regards,
Hui

**Comment by SlongLiu** [comment]
We use famous backbones like ResNet and Swin-Transformer for feature extracting as well in our methods. What we proposed is a framework for multilabel classification other than a new backbone. Our framework is backbone agnostic, that is, our method could be used for any backbone and introduce improvement consistently. 

If you want to extract features only, some traditional backbones like TResNet, ResNet, and Swin-Transformer are suitable enough.   While if you'd like to train a multilabel classifier for your own dataset, you can use our model in your datasets and modify some parameters, like the number of labels, correspondingly.
